newton/tests/test_import_usd.py

```
@@ -322,6 +322,125 @@ def test_filtering(
                 expected_joint_keys=["joint_1", "/Articulation/Joint2", "/Articulation/Joint3"],
             )
 
+    @unittest.skipUnless(USD_AVAILABLE, "Requires usd-core")
+    def test_solimp_friction_parsing(self):
+        """Test that solimp_friction attribute is parsed correctly from USD."""
+        from pxr import Usd  # noqa: PLC0415
```

Ruff (0.14.6)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)